docs/release-notes/v1.1.rst

```diff
@@ -1,6 +1,223 @@
+that will be implemented in the coming feature releases to allow I/O with multiple data formats.
+
+New features
+~~~~~~~~~~~~
+
+*gammapy.data*
+
+- A new function `~gammapy.data.get_irfs_features()` can extract the features of IRFs
+  (such as energy resolution bias or PSF radius) of a list of `Observation`. The output
+  list can then be passed to the function `~gammapy.utils.cluster.hierarchical_clustering()`
+  which will find clusters of IRF features allowing to combine `Observation` with similar
+  IRF characteristics during analysis.
+
+*gammapy.maps*
+
+- A `Map.reorder_axes` helper method has been introduced.
+- Dot product is now supported with the `Map.dot()` function. It applies the dot product on
+  the axes sharing the same name. It can also be called via the `@` operator.
+- A `WcsNDMap.to_region_nd_map_histogram()` helper method is introduced to compute
+  the histogram of the map values along the spatial dimensions for each non spatial axes
+  bins.
+
+*gammapy.modeling*
+
+- The `~gammapy.modeling.models.LightCurveTemplateTemporalModel` now supports energy dependent
+  light curve templates. These models are now created from a `Map` object with a `time` axis
+  and optionally with an `energy_true` axis. They can be read from an `~astropy.table.Table`
+  (only regular light curve models) or from a serialized `Map`. For now, the energy dependent
+  models cannot be used for analysis.
+- A new function `~gammapy.modeling.select_nested_models` has been introduced to perform
+  nested model fits and compute the resulting test statistic (TS) between two nested hypotheses.
+  It can be used to determine whether the addition of a source to the model is significant or
+  to test for specific features in a model (e.g. test the existence of a spectral cutoff).
+ - A new method has been added on `SpectralModel.spectral_index_error()` to compute
+  the spectral index at a given energy as well as its error by error propagation of
+  the covariance matrix elements.
+- The Franceschini (2018) and Saldana-Lopez (2021) EBL are now part of the built-ins
+  EBL models.
+- A spatial correction model can now be added to the `~gammapy.modeling.models.FoVBackgroundModel`.
+  It can be used with a new spatial model, the `~gammapy.modeling.models.PiecewiseNormSpatialModel`.
+
+
+*gammapy.estimators*
+
+- Multiprocessing is now supported for `FluxPointsEstimator` and `LightCurveEstimator`. Setting
+  the number of cores used is done with the `n_jobs` property that is available on these
+  classes.
+
+*gammapy.visualization*
+
+- A function to plot `Map` as RGB is now proposed: `~gammapy.visualization.plot_map_rgb()`.
+- A function to plot the spectral distributions of predicted counts of models defined
+  on a `Dataset` is now available : `~gammapy.visualization.plot_npred_signal()`
+
+*documentation*
+
+- A new tutorial to demonstrate how to use Gammapy with HAWC data.
+- The pulsar analysis tutorial now shows how to perform 3D analyses in phase.
```

Comment:
The pulsar tutorial only shows how to create a `MapDatasetOnOff` and how to extract ON pulse and OFF pulse map. 
It does not show (not yet) how to perform a 3D analysis.